#87 add original + shaded configuration properties

domdorn opened the pull request:
completely replacing the configuration properties resulted in errors when users
where using the unshaded AHC-Client in their applications together with play-ws.
As a workaround we take the original properties and _add_ our own properties to the
file. This way, the unshaded client still finds its properties while our own version can
be configured independently

domdorn commented:
@wsargent should I squash + merge this? would solve the problem imho.

wsargent commented:
@domdorn I don't see any changes in the file here -- only `build.sbt` is modified.

domdorn commented:
@wsargent yes, build.sbt contains the logic for the shading and the rewriting of the ahc.properties file. I just changed:
 - additional to the rewritten properties, the original ones are also added

This basically makes sure, that the classloader will find both variables defined (if it loads the properties from this jar)

wsargent commented:
Ah, okay.
